common_benchmark_suite/openxla/benchmark/comparative_suite/jax_model_definitions.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2023 The OpenXLA Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import string
+
+from openxla.benchmark import def_types, unique_ids
+from openxla.benchmark.comparative_suite import utils
+
+PARENT_GCS_DIR = "https://storage.googleapis.com/iree-model-artifacts/jax/jax_models_0.4.10_1684396752"
+
+# Constants and functions help build batch templates.
+BATCH_ID = lambda model_id: string.Template(model_id + "-batch${batch_size}")
+BATCH_NAME = lambda name: string.Template(name + "_BATCH${batch_size}")
+BATCH_TAG = string.Template("batch-${batch_size}")
+BATCH_SIZE_PARAM = string.Template("${batch_size}")
+
+T5_LARGE_FP32_JAX_IMPL = def_types.ModelImplementation(
+    id=unique_ids.MODEL_T5_LARGE_FP32_JAX,
+    name="T5_LARGE_FP32_JAX",
+    tags=["fp32", "transformer-encoder", "transformer-decoder", "t5"],
+    framework_type=def_types.ModelFrameworkType.JAX,
+    data_type=def_types.ModelDataType.FP32,
```

Comment:
I wonder if data type should be part of the model config? The same implementation can work with different datatypes: https://github.com/iree-org/iree-samples/blob/fda5e8adc1e05646c10b1464063f199427d1c896/iree-torch/library/models/bert_large.py#L15.

Maybe something to revisit later when we port over the PyTorch FP16 models.